test/cypress/tests/ui/DataEmpty.cy.ts

```diff
@@ -0,0 +1,71 @@
+import { createEditorWithTextBlocks } from '../../support/utils/createEditorWithTextBlocks';
+
+describe('inputs [data-empty] mark', function () {
+  it('should be added to the editor on initialization', function () {
+    createEditorWithTextBlocks([
+      'First', // not empty block
+      '', // empty block
+    ]);
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .first()
+      .should('have.attr', 'data-empty', 'false');
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .last()
+      .should('have.attr', 'data-empty', 'true');
+  });
+
+  it('should be added to the block on input', function () {
```

Comment:
Your test case checks that this attr should be toggled on input. Would be great to add a test case that this attr will be set to true if this block is empty on input.

Reply:
added `should be added as "true" to the input on chars removal`